src/components/Cloud/Bandwidth-Usage.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VictoryBar, VictoryChart, VictoryAxis, VictoryTooltip, VictoryLabel } from 'victory';
+
+import DashboardBlock from '../Dashboard-Block';
+import { adminTheme } from '../../victory-theme';
+
+/**
+ * Convert bytes to gigabytes and return a nicely formatted number.
+ *
+ * @param bytes
+ * @returns {string}
+ */
+const convertBytesToGigabytes = bytes => {
```

Comment:
This should go into a utils file if we ever use it again. Something more generic might be useful too that takes a 2nd param like 'k', 'M', 'G' etc

Reply:
For now I moved this to utilities and kept it generic since we don't have a specific need for anything else yet. Should be really easy to modify once we need to though.